pkgs/development/libraries/oracle-instantclient/default.nix

```diff
@@ -59,25 +55,15 @@ let
   srcFilename = component: arch: version: rel:
     "instantclient-${component}-${arch}-${version}" +
     (optionalString (rel != "") "-${rel}") +
-    (optionalString (arch == "linux.x64") "dbru") + # ¯\_(ツ)_/¯
+    (optionalString (arch == "linux.x64" || arch == "macos.x64") "dbru") + # ¯\_(ツ)_/¯
     ".zip";
 
-  # fetcher for the clickthrough artifacts (requiring manual download)
-  fetchClickThrough =  srcFilename: hash: (requireFile {
-    name = srcFilename;
-    url = "https://www.oracle.com/database/technologies/instant-client/downloads.html";
-    sha256 = hash;
-  });
-
   # fetcher for the non clickthrough artifacts
-  fetchSimple = srcFilename: hash: fetchurl {
+  fetcher = srcFilename: hash: fetchurl {
     url = "https://download.oracle.com/otn_software/linux/instantclient/193000/${srcFilename}";
```

Comment:
@flokli just one small thing: `s/linux/mac/`
Aside from that it builds fine on both platforms :+1:

Reply:
This would break fetching for linux.

I guess we'd need a second "lookup table" called `shortArch`, and pass that to `fetcher` as well. Could you update the PR to do that?